docs/GoogleDrive-batchsink.md

```diff
@@ -54,12 +58,57 @@ Service account json can be generated on Google Cloud
   authorization. Can be set to 'auto-detect' when running on a Dataproc cluster.
   When running on other clusters, the file must be present on every node in the cluster.
   
-  When set to 'auto-detect', the cluster needs to be configured with the scope required by the Google Drive API.
+  When set to 'auto-detect', the GCE VM needs to be configured with the scope required by the Google Drive API.
   Otherwise, the preview as well as pipeline run will fail with insufficient permission error. The required scope
-  is "https://www.googleapis.com/auth/drive".
-
-* **JSON**: Contents of the service account JSON file.
-
-Make sure that the Google Drive Folder is shared with the service account email used. 
-To write files to a Private Google Drive, grant the `Editor` role to the specified service account. 
-To write files to a Shared Google Drive Folder, grant the `Contributor` role to the specified service account.
\ No newline at end of file
+  is `https://www.googleapis.com/auth/drive`.
+
+
+* **JSON**: Contents of the service account JSON file. Make sure that the Google Drive Folder is shared with the 
+  service account email used. To write files to a Private Google Drive, grant the `Editor` role to the 
+  specified service account. To write files to a Shared Google Drive Folder, grant the `Contributor` role to the 
+  specified service account.
+
+### Steps to Generate OAuth2 Credentials
+- For the Client ID and Client Secret, you will need create those credentials [here](https://console.cloud.google.com/apis/credentials).
```

Comment:
1. Create credentials for the Client ID and Client Secret properties [here](https://console.cloud.google.com/apis/credentials).
Can we link to the Google cloud topic for creating credentials.